src/CLRunner.ts

```diff
@@ -335,6 +335,110 @@ export class CLRunner {
         return startResponse
     }
 
+    private async StartSession(appId: string, createParams: CLM.SessionCreateParams): Promise<CLM.Session> {
+
+        // Don't save logs on server if custom storage was provided
+        if (ConversationLearner.logStorage) {
+            createParams.saveToLog = false
```

Comment:
>saveToLog [](start = 25, length = 9)

saveToLog param was a model level flag. shouldn't this be decided based on that?

Reply:
The thought was that if you have local storage defined you never save to server storage.  Don't want to write the flag to the server either, as you could load the UI later w/o local storage defined